sessions/composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "google/cloud-firestore": "^1.7"
```

Comment:
This version will need to be bumped once https://github.com/googleapis/google-cloud-php/pull/2258 is merged